Editor/Resources/EditorWindow/Pages/ManagedEC2Page.uxml

```diff
@@ -0,0 +1,46 @@
+            <ui:Label name="ManagedEC2Title" class="page__title" text="Managed EC2"/>
+            <ui:Label name="ManagedEC2Description" text="Managed EC2 fleets use Amazon EC2 instances to host your game servers.&#10;Amazon GameLift manages the instances and removes the burden of hardware and software management from hosting your games."/>
+        </ui:VisualElement>
+        <ui:VisualElement class="divider"/>
+        <ui:Foldout name="ManagedEC2IntegrateTitle" class="foldout separator separator--small" text="Integrate GameLift with your game client">
+            <ui:Label name="ManagedEC2IntegrateDescription" text="Add the Amazon GameLift server SDK to your game server and add code to communicate with the GameLift hosting service. If you&apos;re working with the Amazon GameLift sample game, this integration is already done."/>
+            <ui:VisualElement class="external-link">
+                <ui:Label name="ManagedEC2IntegrateLink" text="Setup Amazon GameLift on a backend service"/>
+                <ui:Image name="ExternalLink" class="icon"/>
+        </ui:VisualElement>
+        </ui:Foldout>
+        <ui:Foldout name="ManagedEC2ScenarioTitle" class="foldout" text="Select Deployment Scenario"/>
+        <ui:Foldout name="ManagedEC2ParametersTitle" class="foldout" text="[GameName] parameters"/>
+        <ui:Foldout name="ManagedEC2DeployTitle" class="foldout separator" text="Deploy (Single-region Fleet)">
+            <ui:Label name="ManagedEC2DeployDescription" class="foldout__description form-row" text="Deploying GameLift can take up to 30-40 minutes"/>
+            <ui:VisualElement class="form-row form-row--free">
+                <ui:Label name="ManagedEC2DeployStatusLabel" class="form-row__label" text="Status"/>
+                <ui:VisualElement name="DeploymentStatus" class="form-row__input deploy-status deploy-status--deployed">
+                    <ui:VisualElement name="ManagedEC2DeployStatusIcon" class="deployStatus test deploy-status__icon"/>
+                    <ui:Label name="ManagedEC2DeployStatusText" class="deploy-status__label" text="State"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+            <ui:VisualElement class="form-row form-row--free">
+                <ui:Label name="ManagedEC2DeployActionsLabel" class="form-row__label" text="AWS Resource Actions"/>
+                <ui:VisualElement class="form-row__input form-row__input--inline separator">
+                    <ui:Button name="ManagedEC2CreateStackButton" class="button button--primary" text="Create"/>
+                    <ui:Button name="ManagedEC2RedeployStackButton" class="button" text="Redeploy"/>
+                    <ui:Button name="ManagedEC2DeleteStackButton" class="button" text="Delete"/>
+                </ui:VisualElement>
+            </ui:VisualElement>
+        </ui:Foldout>
+        <ui:Foldout name="ManagedEC2LaunchClientTitle" class="foldout" text="Launch Client">
```

Comment:
Should the name of this element be `ManagedEC2LaunchClientFoldout` instead?

Reply:
I can change it if needed, I had it as `title` as it does act as the title for that section